### ISSUE TITLE
"Herkunft" und "Befähigung"

### ISSUE THREAD

**Issue by skade** [rubyberlin/code-of-conduct#61]
In #59 gab es einen guten Einwand gegen die Worte "Herkunft" und Befähigung.

https://github.com/rubyberlin/code-of-conduct/pull/59#commitcomment-11303458

Wenn ich das jetzt richtig gelesen habe, ist der generelle Konsens, diese Worte zu ersetzen, auch in allen Übersetzungen, sofern notwendig.

**Comment by skade** [comment]
Ich möchte zu Ethnizität anmerken, dass ich auch schonmal direkt nach CoCs gefragt wurde, die genau das erwähnen und nicht auslassen.

Das ist natürlich ein Problem, dass das Problem direkt benannt werden soll, aber manchmal auch das benennen des Problems problematisch ist. "unabhängig von Geburtsort" klingt ziemlich weich :(.

Zu der Befähigung: können wir da mal bei den Sozialhelden oder Leidmedien nachfragen?

**Comment by hagenburger** [comment]
„Unabhängig vom Geburtsort“: Für viele, die betroffen sein können, ist der Geburtsort ja _hier,_ wie für viele andere auch. Wäre ein Begriff wie „unabhängig vom Äußeren“/„unabhängig der Erscheinung“ hilfreich? Sicher würde der etwas weiter greifen und einen Bereich einschließen, den wir nur indirekt ansprechen.

P. S. diese Diskussion ist sicher auch für Leute interessant, die Deutsch nicht verstehen.

**Comment by 1000miles** [comment]
@skade Ja, das ist gar nicht so einfach, inklusive Communities zu gestalten. :)

Bin auch dafür, dass wir uns Unterstützung holten sollten, da dieser CoC momentan eher von Menschen verfasst/bearbeitet wird, die weniger von Ausschlüssen betroffen sind.

**Comment by lauralindal** [comment]
Wie wäre es vorerst mit "zugeschriebener Ethnizität" und dann machen wir eine weitere Iteration mit Menschen, die da andere Perspektiven reinbringen?

**Comment by skade** [comment]
@1000miles Ich wär da für sammeln und Leute finden.

> da dieser CoC momentan eher von Menschen verfasst/bearbeitet wird, die weniger von Ausschlüssen betroffen sind.

Nun, da einige von diesen Dingen nicht immer einfach sichtbar sind, wär ich mit solchen Aussagen vorsichtig. Ich habe CoCs (und deren Abwesenheit) auch schon aus Anwesenden-Sicht benötigt/ausprobiert :/.

Aber gerade deswegen find ichs wichtig, konkret mit den Menschen zu reden, die von den jeweiligen Ausschlüssen betroffen sind :).

**Comment by 1000miles** [comment]
@skade Huch, Deinen Eintrag irgendwie übersehen. Das, was ich meine, bedarf einer weiteren Ergänzung. Sorry, wenn ich mich nur so knapp ausgedrückt habe. :) Melde mich heute Abend nochmal.

**Comment by 1000miles** [comment]
@skade Du hast natürlich Recht, ich hätte mich besser ausdrücken sollen. Sorry. 
Ich kann natürlich nicht für andere Bereiche wie z.B. "disabilities" sprechen, aber aus einer Position als PoC, die sich viel mit Black und PoC austauscht.

Was ich meinte, ist, alleine durch die Tatsache, dass wir uns in der Tech Community bewegen können, wir deutlich mehr Privilegien gegenüber Menschen haben, die diesen Zugang aus den verschiedensten Gründen nicht haben (daher auch "weniger" und nicht "keine" Ausschlüsse). 

Und mir erscheint als seien einige marginalisierte Gruppen, die ja eigentlich einbezogen werden sollten, bisher beim Code of Conduct noch nicht wirklich zu Wort gekommen sind bzw. deren Perspektiven/Meinungen wurden bisher noch nicht eingeholt.

Von daher deckt sich Deine Idee mit der von Laura und mir, bei so einem wichtigen Dokument wie dem CoC tatsächlich die fehlenden Stimmen im Sinne einer inklusiveren Community einzuholen. :)
